lib/v2/screens/profile_screens/guardians/guardians_tabs/interactor/viewmodels/page_commands.dart

```diff
@@ -11,4 +11,29 @@ class ShowRecoveryStarted extends PageCommand {
   final GuardianModel guardian;
 
   ShowRecoveryStarted(this.guardian);
-}
\ No newline at end of file
+}
+
+class ShowOnboardingGuardianSingleAction extends PageCommand {
+  final int index;
+  final String image;
+  final String description;
+  final String buttonTitle;
+
+  ShowOnboardingGuardianSingleAction(
+      {required this.index, required this.description, required this.image, required this.buttonTitle});
```

Comment:
add a comma after `required this.buttonTitle` and format

Reply:
fix